src/actors/tree/render.inc.c

```diff
@@ -0,0 +1,259 @@
+#include <actors.h>
+#include <PR/gbi.h>
```

Comment:
This this folder `trees` since it appears to be more than just 1 type of tree, yes?

Reply:
yeah it's different tree